### PR TITLE
Fixes typos & broken links (#495)

### PR DESCRIPTION
**Pull request by yadunandanbhat**

### CODE REVIEW

**Review by dgrove-oss — COMMENTED** [review]
Thanks for taking the time to submit the fixes.

File: _layouts/home.html [hunk 1]
```diff
@@ -88,8 +88,8 @@ <h4>Write functions in any language</h4>
                 <a href="https://github.com/apache/openwhisk-runtime-ruby">Ruby</a>,
                 <a href="https://github.com/apache/openwhisk-runtime-rust">Rust</a>,
                 <a href="https://github.com/apache/openwhisk-runtime-java">Scala</a>,
-                <a href="https://github.com/apache/openwhisk-runtime-swift">Swift</a> and even newer, cloud-native ones like
-                There is even an experimental runtime for <a href="https://github.com/apache/openwhisk-runtime-deno">Deno</a> in-development.
+                <a href="https://github.com/apache/openwhisk-runtime-swift">Swift</a> and even newer, cloud-native ones.
```
[the remark this answers is not in the document]
Comment (yadunandanbhat) [reply _layouts/home.html:91]: I'm sorry, I didn't understand??

[the remark this answers is not in the document]
Comment (dgrove-oss) [reply _layouts/home.html:91]: Sorry, I was suggesting you drop the "and even newer, cloud native ones".  So the list becomes just:
```
 <a href="https://github.com/apache/openwhisk-runtime-python">Python</a>,
                <a href="https://github.com/apache/openwhisk-runtime-ruby">Ruby</a>,
                <a href="https://github.com/apache/openwhisk-runtime-rust">Rust</a>,
                <a href="https://github.com/apache/openwhisk-runtime-java">Scala</a>,
                <a href="https://github.com/apache/openwhisk-runtime-swift">Swift</a>.
                There is also an experimental runtime for <a href="https://github.com/apache/openwhisk-runtime-deno">Deno</a> in-development.
```

[the remark this answers is not in the document]
Comment (yadunandanbhat) [reply _layouts/home.html:91]: ohhh now i get it! sorry english isn't my first language so i mess up sometimes! i'll update it and push a commit!

**Pull request merged**